sip/headers.go

```diff
@@ -407,6 +417,28 @@ func (hs *headers) RecordRoute() *RecordRouteHeader {
 	return hs.recordRoute
 }
 
+// ReferTo returns underlying Refer-To parsed header or nil if not exists
+func (hs *headers) ReferTo() *ReferToHeader {
```

Comment:
you can keep lazy parsing, but you need to remove this fast references.

Reply:
do you mean automatic assignment to h.referTo/h.referredBy need to be removed? I don't get what you mean by "fast references"

Reply:
yes.